broke-protocol/src/main/java/org/dsngroup/broke/protocol/Message.java

```diff
@@ -36,9 +36,9 @@
     // Message fields
     // TODO: may let this message to release the burden of construction
     // and destruction.
-    protected Method method;
-    protected MessageHeader header;
-    protected Object payload;
+    protected final Method method;
+    protected final MessageHeader header;
+    protected final Object payload;
 
```

Comment:
I'm wondering if further we can eliminate the construction and destruction burden from these fields. Maybe `final` comes with the lower flexibility.

Reply:
OK, I'll remove the ```final``` keyword.